hack/generate-versions.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+#
+# A naive way to generate `jsonnet/versions.json` file from downstream forks
+# It uses `VERSION` file located in each repository or a special function to figure out correct version
+# 
+# Script is based on https://github.com/prometheus-operator/kube-prometheus/blob/main/scripts/generate-versions.sh
+#
+
+get_version_from_file() {
+  curl --retry 5 --silent --fail "https://raw.githubusercontent.com/${1}/master/VERSION"
```

Comment:
Since we are always getting the version from the master branch, does it mean we should only run this command for new releases and not for backports?

Reply:
We lock versions during release preparation and (mostly) do not update them for backports. As such I didn't see a need to complicate script for this.